lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -246,12 +254,41 @@ class PatternStopContents extends Component<Props, State> {
     updatePatternStops(activePattern, patternStops)
   }
 
+  _onPickupOrDropOffChange = (evt: SyntheticInputEvent<HTMLInputElement>) => {
+    const selectedOptionValue: number = parseInt(evt.target.value)
```

Comment:
Always supply the `radix` argument to `parseInt`. Learn more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#octal_interpretations_with_no_radix).